modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryVersionLocalServiceImpl.java

```diff
@@ -48,7 +50,20 @@ public ObjectEntryVersion deleteObjectEntryVersion(
 			long objectEntryId, int version)
 		throws PortalException {
 
-		// TODO VALIDATION
+		if (getObjectEntryVersionsCount(objectEntryId) == 1) {
```

Comment:
If I understand correctly, both validations cover the same thing, so just one of them would be enough, right?